front/credential.form.php

```diff
@@ -31,11 +31,21 @@
  * ---------------------------------------------------------------------
  */
 
-include("../../../inc/includes.php");
-
+include('../../../inc/includes.php');
 $dropdown = new PluginGlpiinventoryCredential();
-include(GLPI_ROOT . "/front/dropdown.common.form.php");
 
-if (isset($_GET['popup'])) {
-    Ajax::refreshDropdownPopupInMainWindow();
+if (!empty($_POST) || isset($_GET['_in_modal'])) {
+    include(GLPI_ROOT . "/front/dropdown.common.form.php");
+} else {
+    Html::header(
+        __('GLPI Inventory', 'glpiinventory'),
+        $_SERVER["PHP_SELF"],
+        "admin",
+        "pluginglpiinventorymenu",
+        "credential"
+    );
+    PluginGlpiinventoryMenu::displayMenu("mini");
+    //If there is no form to submit, display the form
+    $dropdown->display($_GET);
+    Html::footer();
 }
```

Comment:
Not sure it is used, but it was here previously.
```suggestion
}

if (isset($_GET['popup'])) {
    Ajax::refreshDropdownPopupInMainWindow();
}
```

Reply:
done